conf/gulp-tasks/library.gulpfile.js

```diff
@@ -19,6 +20,23 @@ const uglify = require('gulp-uglify-es').default;
 
 const NAMESPACE = 'ANSWERS';
 
+function getLibVersion () {
+  const hash = require('child_process')
+    .execSync('git rev-parse HEAD')
+    .toString().trim();
+
+  const version = require('child_process')
+    .execSync('git describe --match "v[0-9]*" --abbrev=0 HEAD --always')
+    .toString().trim();
+
+  let libVersion = `canary-${hash}`;
+  if (hash !== version) {
+    libVersion = `${version}`;
```

Comment:
nit but could we just return here version !== hash and below, and get rid of the libVersion variable? ${version} can also just be version

Reply:
done